substrate/frame/session/src/migrations/v1.rs

```diff
@@ -49,14 +49,14 @@ impl<T: Config> MigrateDisabledValidators for InitOffenceSeverity<T> {
 	fn peek_disabled() -> Vec<(u32, OffenceSeverity)> {
 		DisabledValidators::<T>::get()
 			.iter()
-			.map(|v| (*v, OffenceSeverity(Perbill::zero())))
+			.map(|v| (*v, OffenceSeverity::default()))
```

Comment:
For example here `default()` doesn't mean much to the readed in comparison with `most_severe()`.